### PR TITLE
Update CyhyMessage.py

### PR DESCRIPTION
**Pull request by jeffkause**
#  #

## 🗣 Description ##

To reduce report question emails, add language to the weekly report email messages(CyHyMessage.py) explaining where and how to access CSV files in  Appendix G of the CyHy report.

## 💭 Motivation and context ##

The email change will reduce VS InBox emails.

## 🧪 Testing ##

The additional language will appear in the weekly report emails.

image

```console
$ pytest
======================================================================== test session starts ========================================================================
platform darwin -- Python 3.12.5, pytest-8.3.2, pluggy-1.5.0
rootdir: /Users/XXXXX/cyhy-mailer
plugins: cov-5.0.0
collected 31 items                                                                                                                                                  

tests/test_cybexmessage.py ..                                                                                                                                 [  6%]
tests/test_cyhymessage.py .....                                                                                                                               [ 22%]
tests/test_cyhynotificationmessage.py ........                                                                                                                [ 48%]
tests/test_httpsmessage.py ....                                                                                                                               [ 61%]
tests/test_message.py ....                                                                                                                                    [ 74%]
tests/test_reportmessage.py ..                                                                                                                                [ 80%]
tests/test_statsmessage.py ..                                                                                                                                 [ 87%]
tests/test_tmailmessage.py ....                                                                                                                               [100%]

======================================================================== 31 passed in 0.11s =========================================================================
```

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [X] All new and existing tests pass.

### CODE REVIEW

File: cyhy/mailer/CyhyMessage.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ class CyhyMessage(ReportMessage):
 
 <p>The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)</p>
 
+<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.
```
Comment (jsf9k) [inline cyhy/mailer/CyhyMessage.py]: I'm pretty sure "paperclip" is a single word:
```suggestion
<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paperclip icon to the left of the attachment name.
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
You also need to make the `pytest` tests pass.  Interestingly, you checked the "Tests have been added..." box even though you didn't touch a single test!

File: cyhy/mailer/CyhyMessage.py (hunk 1 above)
Comment (jeffkause) [reply cyhy/mailer/CyhyMessage.py]: Looking at the report pdf, the word is spelled paper clip. Do we want to keep it consistent with the report?

Comment (jsf9k) [reply cyhy/mailer/CyhyMessage.py]: Yes, I suppose we do.

File: cyhy/mailer/CyhyMessage.py [hunk 2]
```diff
@@ -61,6 +61,8 @@ class CyhyMessage(ReportMessage):
...
+
```
Comment (dav3r) [inline cyhy/mailer/CyhyMessage.py:67]: @jeffkause You also need to add this new verbiage (minus the HTML) to the `TextBody` section above:
https://github.com/cisagov/cyhy-mailer/blob/8a80b7066d3273188d01708c49721c9e6db4bbb5/cyhy/mailer/CyhyMessage.py#L32-L35

**Review by dav3r — CHANGES_REQUESTED** [review]

File: cyhy/mailer/CyhyMessage.py (hunk 2 above)
Comment (jeffkause) [reply cyhy/mailer/CyhyMessage.py:67]: I have completed the additional wording and completed a successful pytest.

Comment (jsf9k) [reply cyhy/mailer/CyhyMessage.py:67]: Don't forget to push up your changes!

Comment (jeffkause) [reply cyhy/mailer/CyhyMessage.py:67]: Pushed. TY

File: cyhy/mailer/CyhyMessage.py [hunk 3]
```diff
@@ -61,6 +63,8 @@ class CyhyMessage(ReportMessage):
 
 <p>The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)</p>
 
+<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.
```
Comment (jsf9k) [inline cyhy/mailer/CyhyMessage.py]: You should really close the `<p>` tag:
```suggestion
<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.</p>
```

You will need to update the test code accordingly.

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: cyhy/mailer/CyhyMessage.py (hunk 2 above)
Comment (dav3r) [reply cyhy/mailer/CyhyMessage.py:67]: Looks like the `TextBody` verbiage was included in https://github.com/cisagov/cyhy-mailer/pull/111/commits/c7fc84d1c68bfde772d4eae4ea2f8818a54ad460.

@jeffkause In the future, please:
* Mention the commit URL (https://github.com/cisagov/cyhy-mailer/pull/111/commits/c7fc84d1c68bfde772d4eae4ea2f8818a54ad460) when you respond to the conversation; this makes it easier to see where you addressed the concerns raised in the conversation
* Replace the default GitHub commit message ("Update CyhyMessage.py") with a more useful commit message (e.g. "Add text-only content matching HTML email content").

File: cyhy/mailer/CyhyMessage.py (hunk 3 above)
Comment (jsf9k) [reply cyhy/mailer/CyhyMessage.py]: Also, please use more descriptive commit messages than "Update filename.py".  Describe what changes you're actually making.

Comment (jsf9k) [reply cyhy/mailer/CyhyMessage.py]: Done in commit 2af19ca.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

**Pull request merged**